tests/BackupManager/Databases/MysqlDatabaseTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace DigipolisGent\Tests\Robo\Task\Deploy\BackupManager\Databases;
+
+use DigipolisGent\Robo\Task\Deploy\BackupManager\Databases\MysqlDatabase;
+
+class MysqlDatabaseTest extends \PHPUnit_Framework_TestCase
```

Comment:
If the behaviour changes to be more generic, so to allow both tables and structuretables to be set, we'll need another testcase in here